Formula/glbinding.rb

```diff
@@ -11,12 +11,22 @@ class Glbinding < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "glfw3" => :optional
   needs :cxx11
 
   def install
+    args = std_cmake_args.clone
+
+    if build.with? "glfw3"
+      args << "-DOPTION_BUILD_TOOLS=ON"
+    else
+      ohai "You are building without tool support. To enable tools, reinstall with 'brew reinstall glbinding --with-glfw3'"
```

Comment:
Don't `ohai` in formula. The extreme majority of users will never see the warning because they'll pour the bottle.